src/cloudant/client.py

```diff
@@ -775,7 +776,10 @@ def bluemix(cls, vcap_services, instance_name=None, **kwargs):
 
             print client.all_dbs()
         """
-        service = CloudFoundryService(vcap_services, instance_name)
+        service_name = service_name or 'cloudantNoSQLDB'  # default service
```

Comment:
Moved `cloudantNoSQLDB` default value out of the `CloudFoundryService` class as it's specific to Bluemix.